models/models.js

```diff
@@ -35,8 +35,17 @@ exports.selectArticles = (sort_by, order) => {
         "article_img_url"
     ]
     const validOrders = ["asc", "desc"]
-    if(sort_by && order){
-        if(!validOrders.includes(order) || !validColumns.includes(sort_by)){
+    const validTopics = ["mitch", "cats", "paper"]
```

Comment:
Ideally The existence of the topic should come from the db, not to be hard coded so it remains up to date